KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+
+## Motivation
+To enhance individual voting power and promote ultimate decentralization, we aim to accomplish an equal proposer selection structure. With the change of the voting method, GC members who share the same interest and values as Klaytn can gain more power and influence by staking and locking up more KLAY.
+
+## Specification
+
+The Klaytn governance voting system is designed based on the following fundamental premises. 
+- We believe Klaytn’s major decision-making process should reflect the opinions of as many participants as possible from the ecosystem. 
+- We assumed that participants are more likely to make a decision that is beneficial to the Klaytn ecosystem if they hold more KLAY. This is based on the premise that the growth of Klaytn’s ecosystem is correlated with the rise in the value of KLAY. 
+- The governance system should be able to manage the situations in which a particular entity makes an arbitrary decision. This is because the entity may weaken the sustainability of the entire network. 
+- The act of voting and the gain of voting power is different. 
+
+Governance Council members can exercise the right to vote based on the staking amount. Currently, each GC member receives at least 1 voting right regardless of the number of KLAY they hold. Recognizing a tendency that the member with more KLAY makes a decision that benefits the Klaytn ecosystem in the long run, we are granting a higher voting power to members with more KLAY. Yet, to prevent a particular subject from making an arbitrary decision, we are placing a cap on the voting power one can receive. 
+Therefore, the GC member will receive 1 vote per a certain amount of staked KLAY (initial configuration: 500 million KLAY). The maximum number of votes a GC member can own is one less than the total number of members. In other words, [Maximum Voting Power =  Total number of GC members - 1]. For example, if there are 35 GC members, one can own a maximum of 34 voting power. The 1 KLAY: 1 vote structure prevents the tragedy of the commons and potential monopolies by limiting the maximum number of votes cast. 
+The foundation will provide `votingDelay` (initial configuration: 7 days)  of the notice period and ‘votingPeriod’ of the voting period (initial configuration: 7 days). The list of eligible voters is finalized at the start of the notice period. The qualified electors and voting power of those will be counted at the start of the voting period. The foundation or GC can freely propose and discuss on the forum. Once the discussion is held on the forum, the foundation will put the topic to a vote on the governance portal. 
```

Comment:
```suggestion
The foundation will provide `votingDelay` (initial configuration: 7 days) of the notice period and `votingPeriod` of the voting period (initial configuration: 7 days). The list of eligible voters is finalized at the start of the notice period. The qualified electors and voting power of those will be counted at the start of the voting period. The foundation or GC can freely propose and discuss on the forum. Once the discussion is held on the forum, the foundation will put the topic to a vote on the governance portal. 
```

Reply:
1. It could be better to have a diagram to describe `votingDelay` and `votingPeriod`. Is it possible to overlap the two period?
2. In the diagram, it would be good if we specify when the list of eligible voters are finalized.

Reply:
@kjhman21 
1. I will add a linear diagram describing all successful states (pending to execution) with their respective periods.
  - No they do not overlap. They are consecutive. `votingDelay` defines the Pending period and `votingPeriod` defines the Active period.
2. Yes the diagram will contain both "voter finalizing time" and "voting power finalizing time".
3. I don't think we have an explicit list of allowed topics. Should we add a few examples?

Reply:
@kjhman21
1, 2. Please refer to the latest "Voting contract" section.
3. Yes. First, discuss on Forum, then on-chain voting on Portal. Refer to the "Specification" section.